src/Symfony/Component/Console/Tests/Helper/QuestionHelperTest.php

```diff
@@ -908,6 +909,26 @@ public function testAskThrowsExceptionOnMissingInput()
         $dialog->ask($this->createStreamableInputInterfaceMock($this->getInputStream('')), $this->createOutputInterface(), new Question('What\'s your name?'));
     }
 
+    /**
+     * @expectedException        \Symfony\Component\Console\Exception\RuntimeException
+     * @expectedExceptionMessage Aborted
```

Comment:
Aborted.

Missing dot at the end?
Same below

Reply:
Not critical as `@expectedExceptionMessage()` behaves like `assertContains()`, but done :)